LoopKitUI/Resources/Assets.xcassets/WarningOrange.colorset/Contents.json

```diff
@@ -0,0 +1,38 @@
+{
+  "colors" : [
```

Comment:
Hmm, we override these colors on an app basis, I think. Why are they in LoopKit? I think if they're in LoopKit, then the overriding system won't work.